src/api/v2/agent-templates/services/revalidate-dashboard.ts

```
@@ -0,0 +1,148 @@
+ *
+ * Contract:
+ *   - Best-effort: never throws. Errors are logged at warn and swallowed.
+ *   - 3-second hard timeout via AbortController so a slow dashboard cannot
+ *     stall an HTTP handler that fires this from its hot path.
+ *   - The dashboard's 60s TTL is the fallback when this call drops.
+ */
+
+import {
+  BUILDER_REVALIDATE_SECRET,
+  BUILDER_SITE_URL,
+} from "@/config";
```

ESLint

[error] Replace `⏎··BUILDER_REVALIDATE_SECRET,⏎··BUILDER_SITE_URL,⏎` with `·BUILDER_REVALIDATE_SECRET,·BUILDER_SITE_URL·`

(prettier/prettier)